src/docs/implementations/dynatrace.adoc

```diff
@@ -106,8 +104,10 @@ management.metrics.export.dynatrace:
 
 When the API version is configured to "v2", the registry will send data using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Metrics API v2].
 The v2 exporter does not require any properties to be set except for the version.
-In this case, metrics will be exported to the local OneAgent endpoint.
-If no OneAgent is running on the target host, it is possible to specify an endpoint and an API token in order to export metrics to that specific endpoint.
+When running Micrometer without Spring Boot, the default version is `v1` in order to maintain backwards compatibility.
+The version is assumed to be `v2` automatically when using Spring Boot, and does not have to be set explicitly in that case.
```

Comment:
Would it make sense to change the logic here as well to use v2 if there's no deviceId set?